README.md

```diff
@@ -31,8 +31,38 @@ When the reader has completed this Code Pattern, they will understand how to:
+
+PouchDB, CouchDB, and Service Worker are the primary tools that turn our simple shopping list app into a high performance, offline-capable Progressive Web App.
+
+**Data stays safe on your device, even while it's offline.**  <br>Persistance of shopping lists and item data entered by the user is achieved using the in-browser database PouchDB. This will allow your data to survive between sessions and when disconnected from the network. (Whether you remember that you need juice while you're on your trusty home WiFi or in the middle of the wilderness, you can still add it your list.)
+
+**Data syncs between devices when a connection is available.**  <br>When a connection is available, the data is synced from the local device to a CouchDB database in the cloud, and can thus be shared across multiple devices or users. (Need to share your grocery list with your roommate or access it on both your phone and your laptop? No problem!)
+
+**The app loads quickly, even while offline.**  <br>To keep the app itself functional while offline, a Service Worker is used to cache page resources (the most important HTML, CSS, and JavaScript files) when the web application is first visited. Each device must have a connection for this first visit, after which the app will be fully functional even while offline or in shoddy network conditions. (No more error messages or frustratingly slow page loads.)
+
+**The app can be installed on a mobile device.**  <br>In combination with the Service Worker used for caching, a manifest file containing metadata allows the app to become a Progressive Web App, an enhanced website that can be installed on a mobile device and can then be used with or without an internet connection. (It's secretly still a website, but you can access it through one of those handy dandy little app icons on your homescreen!)
+
+Explore the code in this GitHub repository to see how the Offline First design is applied.
+
+# Tutorial
+
+Refer to the [tutorial](https://github.com/ibm-watson-data-lab/shopping-list-vanillajs-pouchdb/tree/master/tutorial) for step-by-step instructions on how to build your own Offline First shopping list Progressive Web App with Vanilla JS (aka plain old JavaScript) and PouchDB.
+
+# Live demo 
+
+To see this app in action without installing anything, simply visit https://ibm-watson-data-lab.github.io/shopping-list-vanillajs-pouchdb/ in a web browser or on your mobile device.
+
 # Steps
 
+Want to check out the end product on your own machine? Follow these steps to deploy your own instance of the shopping list app.
+
+**THIS PARAGRAPH NEEDS HELP FROM SOMEONE WHO UNDERSTANDS THESE OPTIONS.** <BR>If X, you may want to deploy the app to the IBM Cloud. Alternatively, you may run it locally using your own CouchDB instance or the hosted Cloudant service. 
```

Comment:
Maybe:

>This app can be deployed to IBM Cloud. You can also run this app on your local machine for development purposes using either a local Apache CouchDB instance or an IBM Cloudant service instance from the IBM Cloud Catalog.

Reply:
This still doesn't help me understand what the options mean. 

What's the difference between the first sentence and the second half of the last sentence? In the first your HTML file lives on Bluemix and you use Cloudant, and in the latter your HTML file is on your laptop and you use Cloudant?